MrTheGood/projectb_android
# Multiple themes

Changes:
 * Added Dark theme
 * Added Purple theme
 * Bottom navigation view is now always labeled.

How to test: 
 1. Change the theme
 2. Confirm that it is saved even after force stopping the app
 3. Confirm that every page is completely visible 

Screenshots:
![device-2019-04-02-171721](https://user-images.githubusercontent.com/23239376/55417330-ce754200-5570-11e9-8e97-897bbd06acad.png)
![device-2019-04-02-171758](https://user-images.githubusercontent.com/23239376/55417331-ce754200-5570-11e9-9049-03acf534427d.png)
![device-2019-04-02-172133](https://user-images.githubusercontent.com/23239376/55417332-ce754200-5570-11e9-9e5d-2dcc35f21697.png)
![device-2019-04-02-175811](https://user-images.githubusercontent.com/23239376/55417423-f369b500-5570-11e9-9523-7a43aff02ff4.png)
![device-2019-04-02-175831](https://user-images.githubusercontent.com/23239376/55417425-f4024b80-5570-11e9-96c0-96e91349a645.png)
